Sources/ArcGISToolkit/FloorFilter/FloorFilterViewController.swift

```diff
@@ -0,0 +1,483 @@
+        floorFilterView.layer.shadowOpacity = 0.8
+    }
+    
+    /// Add a corner radius for the cells in the levels table.
+    /// Depending on the visible state of the floor filter.
+    private func setTableViewCellCornerRadius(cell: UITableViewCell) {
+        if state == .fullyExpanded {
+            cell.cornerRadius(usingCorners: [.allCorners], cornerRadii: .zero)
+            
+        } else {
+            if (style == .down) {
+                cell.cornerRadius(usingCorners: [.bottomLeft, .bottomRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+            } else {
+                cell.cornerRadius(usingCorners: [.topLeft, .topRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+            }
+        }
+    }
+    
+    /// Add a corner radius to the site and close button.
+    /// Depending on the placement of the Floor Filter and the current state.
+    private func addCornerRadiusBasedOnPlacement() {
+        if (style == .down) {
+            switch state {
+            case .fullyExpanded:
+                siteBtn?.cornerRadius(usingCorners: [.topLeft, .topRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+                closeBtn?.cornerRadius(usingCorners: [.bottomLeft, .bottomRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+            
+            case .partiallyExpanded:
+                siteBtn?.cornerRadius(usingCorners: [.topLeft, .topRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+                
+            case .initiallyCollapsed:
+                siteBtn?.cornerRadius(usingCorners: .allCorners, cornerRadii: CGSize(width: 5.0, height: 5.0))
+            }
+        } else {
+            switch state {
+            case .fullyExpanded:
+                siteBtn?.cornerRadius(usingCorners: [.bottomLeft, .bottomRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+                closeBtn?.cornerRadius(usingCorners: [.topLeft, .topRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+            
+            case .partiallyExpanded:
+                siteBtn?.cornerRadius(usingCorners: [.bottomLeft, .bottomRight], cornerRadii: CGSize(width: 5.0, height: 5.0))
+                
+            case .initiallyCollapsed:
+                siteBtn?.cornerRadius(usingCorners: .allCorners, cornerRadii: CGSize(width: 5.0, height: 5.0))
+            }
+        }
+    }
+    
+    /// Logic to adjust the floor filter if it placed on the top or bottom of the screen.
+    private func adjustConstraintsBasedOnPlacement() {
+        guard let floorFilterStackView = floorFilterStackView else { return }
+        addCornerRadiusBasedOnPlacement()
+        
+        // First remove the Close Button and Site Button from the stack and then based on placement add the Buttons.
+        floorFilterStackView.removeArrangedSubview(closeBtn)
+        floorFilterStackView.removeArrangedSubview(siteBtn)
+        floorFilterStackView.setNeedsLayout()
+        floorFilterStackView.layoutIfNeeded()
```

Comment:
This line is causing the following to be logged to the console:

```
[TableView] Warning once only: UITableView was told to layout its visible cells and other contents without being in the view hierarchy (the table view or one of its superviews has not been added to a window). This may cause bugs by forcing views inside the table view to load and perform layout without accurate information (e.g. table view bounds, trait collection, layout margins, safe area insets, etc), and will also cause unnecessary performance overhead due to extra layout passes. Make a symbolic breakpoint at UITableViewAlertForLayoutOutsideViewHierarchy to catch this in the debugger and see what caused this to occur, so you can avoid this action altogether if possible, or defer it until the table view has been added to a window.
```

Please investigate and fix.